hack/remove-members.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var dryrun bool
+var configPath string
+
+func main() {
+	flag.StringVar(&configPath, "path", ".", "Path to config directory/subdirectory")
+	flag.BoolVar(&dryrun, "dryrun", true, "Enable Dryrun or not. This flag controls whether the changes are simulated or live. If the changes are simulated it will only print the removal details and the commit message")
```

Comment:
@mrbobbytables for feedback on the help text

Reply:
It could probably be simplified down to  something like: `Simulates changes to be applied and prints removal details`